README.md

```diff
@@ -16,6 +16,14 @@ or
 
 To serve the storybook just run `npm run storybook` which will serve locally on your machine via localhost.
 
+## Design
+
+For the most part all designs laid out in this component library are spearheaded by Vivy design - therefore we take
+a lot of our design cues directly from them. In case a design does not exist, grab a designer and talk over the implementation
+details with them. You can always create a quick placeholder - but official design must be signed off and accepted by the design team.
+
+You can find all current, updated, and new designs in the [Vivy zeplin web project](https://app.zeplin.io/project/5a8d80ddc1a5d9014b9a9b29)
```

Comment:
Is that link authenticated?

Reply:
I believe you need to sign in with zeplin in order to see it, should I redact the project/id from the url?

Reply:
Yea, probably better to be not so transparent

Reply:
Cool